config/routes.rb

```diff
@@ -1,5 +1,6 @@
 Rails.application.routes.draw do
   devise_for :users
   root to: "home#index"
+  get 'about' => 'about#show'
```

Comment:
I meant it should be:
`get "about", to: "about#show"` instead of `get "about" => "about#show"`
sorry for the confusion